be/src/fs/fs_s3.cpp

```diff
@@ -421,7 +502,17 @@ StatusOr<std::unique_ptr<RandomAccessFile>> S3FileSystem::new_random_access_file
         return Status::InvalidArgument(fmt::format("Invalid S3 URI: {}", path));
     }
     auto client = new_s3client(uri, _options);
-    auto input_stream = std::make_unique<io::S3InputStream>(std::move(client), uri.bucket(), uri.key());
+    int64_t read_ahead_size = 64 * 1024; // default value is 64KB
+    auto it = _options._fs_options.find("fs.s3a.readahead.range");
+    if (it != _options._fs_options.end() && !it->second.empty()) {
+        try {
+            read_ahead_size = std::stoi(it->second);
+        } catch (std::logic_error const&) {
+            LOG(WARNING) << " Can not convert config fs.s3a.readahead.range's value to int : " << it->second;
```

Comment:
if a fs_options is provided with invalid range arg, here it will fallback to the default value 64KB, and continue read the data with no issue. However, due to the frequency of using this interface, this WARNING logline will be printed a lot in the be warning log. That will be just noisy. Consider LOG_EVERY_N or down to debug or whatever.

Reply:
yes